docs/getting_access/eessi_wsl.md

```diff
@@ -0,0 +1,118 @@
+Install using 'wsl.exe --install <Distro>'.
+
+NAME                                   FRIENDLY NAME
+Ubuntu                                 Ubuntu
+Debian                                 Debian GNU/Linux
+kali-linux                             Kali Linux Rolling
+Ubuntu-18.04                           Ubuntu 18.04 LTS
+Ubuntu-20.04                           Ubuntu 20.04 LTS
+Ubuntu-22.04                           Ubuntu 22.04 LTS
+Ubuntu-24.04                           Ubuntu 24.04 LTS
+OracleLinux_7_9                        Oracle Linux 7.9
+OracleLinux_8_7                        Oracle Linux 8.7
+OracleLinux_9_1                        Oracle Linux 9.1
+openSUSE-Leap-15.5                     openSUSE Leap 15.5
+SUSE-Linux-Enterprise-Server-15-SP4    SUSE Linux Enterprise Server 15 SP4
+SUSE-Linux-Enterprise-15-SP5           SUSE Linux Enterprise 15 SP5
+openSUSE-Tumbleweed                    openSUSE Tumbleweed
+```
+
+### List the installed machines
+
+```
+C:/users/user>wsl --list --verbose
+  NAME      STATE           VERSION
+* Debian    Stopped         2
+```
+
+### Reconnecting to a Virtual machine with wsl
+
+```
+C:/users/user>wsl --distribution Debian
+user@id:~$
+```
+
+For more documentation on using WSL you can check out the following pages:
+
+* [Install WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+
+* [Basic commands for WSL](https://learn.microsoft.com/en-us/windows/wsl/basic-commands)
+
+## Installing a linux distribution with WSL
+
+```
+C:/users/user>wsl --install --distribution Debian
+Debian GNU/Linux is already installed.
+Launching Debian GNU/Linux...
+Installing, this may take a few minutes...
+Please create a default UNIX user account. The username does not need to match your Windows username.
+For more information visit: https://aka.ms/wslusers
+Enter new UNIX username: user
+New password:
+Retype new password:
+passwd: password updated successfully
+Installation successful!
+```
+
+## Installing EESSI in the Virtual machine
+
+``` { .bash .copy }
+# Installation commands for Debian-based distros like Ubuntu, ...
+
+# install CernVM-FS
+sudo apt-get install lsb-release
+sudo apt-get install wget
+wget https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest_all.deb
+sudo dpkg -i cvmfs-release-latest_all.deb
+rm -f cvmfs-release-latest_all.deb
+sudo apt-get update
+sudo apt-get install -y cvmfs
+
+# install EESSI configuration for CernVM-FS
+wget https://github.com/EESSI/filesystem-layer/releases/download/latest/cvmfs-config-eessi_latest_all.deb
+sudo dpkg -i cvmfs-config-eessi_latest_all.deb
+
+# create client configuration file for CernVM-FS (no squid proxy, 10GB local CernVM-FS client cache)
+sudo bash -c "echo 'CVMFS_CLIENT_PROFILE="single"' > /etc/cvmfs/default.local"
+sudo bash -c "echo 'CVMFS_QUOTA_LIMIT=10000' >> /etc/cvmfs/default.local"
+
+# make sure that EESSI CernVM-FS repository is accessible
+sudo cvmfs_config setup
+
+# start CernVM-FS on WSL
+sudo cvmfs_config wsl2_start
```

Comment:
This should probably be separated out (or repeated) as it needs to run every time you start up the VM

Reply:
I haven't tried it myself, but if I remember correctly, Kurt mentioned in the last monthly meeting that this was no longer required and you could just start autofs. Also see: https://klust.github.io/windows-client-HPC/4_Cluster_Stack/4_01_EESSI/#example-setup-on-fedora-remix-in-wsl2.

Reply:
Wanted to test the fedora remix does not seem to be free. But I can see if can do the same with one of the other wsl standard distributions.

Reply:
And this should indeed be separate. but this only needs to be redone if the vm is terminated and than restarted. I will add this to the documentation.

Reply:
I'll also look at the other thing maybe its better to go that route. I'll test it and than update the docs

Reply:
I think it is still needed with Ubuntu, but not with OpenSUSE

Reply:
There is also an Ubuntu example on the page that Bob shared

Reply:
I put it separately and add a link to how to set up the automounter. I've also tested it for `OpenSUSE-Tumbleweed` and `Ubuntu-24.04`. Which both were easy to do.